tests/test_otpd.py

```diff
@@ -9,7 +10,7 @@ def test_hf_otpd():
     mol = compound.xyz_to_mol(path+'/data/H2O.xyz', 'def2svp', charge=0, spin=0)
     dm = fields.dm.get_converged_dm(mol, xc="pbe")
     otpd, grid = fields.hf_otpd.hf_otpd(mol, dm, return_all=True)
-    assert(abs(otpd @ grid.weights-20.190382555868)<1e-10)
```

Comment:
I think I somehow made pyscf 2.5.0 compatible with qstack with this one line

Reply:
yeah look at those numbers:
```
print(otpd @ grid.weights, 20.190382555868)
20.19039044258982 20.190382555868
```

Reply:
mmh not sure it really solves compatibility, I think the threshold for accuracy is just lower with `np.allclose()` 
but I guess it makes a better test!
The assertion error in this case was not so far off anyway
```
 assert 7.886721846261935e-06 < 1e-10
```
_using `pyscf=2.2.0`_